dali/test/python/test_operator_crop.py

```diff
@@ -505,4 +505,4 @@ def test_slice_with_out_of_bounds_error():
     for device in ['gpu', 'cpu']:
         for batch_size in [1, 3]:
             yield check_crop_with_out_of_bounds_error, \
-                device, batch_size, in_shape
\ No newline at end of file
+                device, batch_size, in_shape
```

Comment:
Missing newline

Reply:
Quite the opposite, I think. There was a missing line before, I added it :)

Reply:
True :)